src/kirby/components/chip/chip.component.scss

```
@@ -0,0 +1,17 @@
+@import "../../scss/web-imports";
+$min-chip-width: 50px;
+.chip {
```

No need to wrap in additional layer, just use `:host` styling:
```suggestion
:host {
```